res/themes/light/css/_light.scss

```diff
@@ -14,7 +14,7 @@ $monospace-font-family: Inconsolata, Twemoji, 'Apple Color Emoji', 'Segoe UI Emo
 
 // unified palette
 // try to use these colors when possible
-$accent-color: #0DBD8B;
+$accent-color:  #00818b;
```

Comment:
Changes to this file need to be reverted, so we don't change the default behaviour of element.  Instead these changes go into our copy of this file in aether-host-files

Reply:
done